### ISSUE TITLE
Apply styles to images

### ISSUE THREAD

**Issue by withoutboats** [withoutboats/notty#40]
Images currently ignore styles. While (most) styles cannot be applied to images, this is not 100% correct. Specifically:
- The opacity style in the cell containing the image data _should_ be applied to the image as a whole.
- The background color of every cell the image covers _should_ be set, even though they will probably not be visible.

**Comment by nokaa** [comment]
I'm not entirely sure how images work with notty; but background color could come into play if the image has transparency like some pngs and gifs do.

**Comment by withoutboats** [comment]
Exactly. Transparent images are why it matters that the background is set correctly.
